lib/linguist/languages.yml

```diff
@@ -6648,3 +6648,13 @@ xBase:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 421
+Carp:
+  type: programming
+  ace_mode: clojure
+  codemirror_mode: clojure
+  codemirror_mime_type: text/x-clojure
+  color: "#c63c24"
```

Comment:
For convenience: The color is `#c63c24`

Reply:
What's the markdown for embedding color?